src/main/java/hudson/os/windows/ManagedWindowsServiceLauncher.java

```diff
@@ -453,7 +454,7 @@ protected String determineHost(Computer c) {
     
     private String createAndCopyJenkinsSlaveXml(String java, String serviceId, PrintStream logger, SmbFile remoteRoot) throws IOException {
         logger.println(Messages.ManagedWindowsServiceLauncher_CopyingSlaveXml(getTimestamp()));
-        String xml = generateSlaveXml(serviceId,
+        String xml = generateSlaveXml(getClass(), serviceId,
```

Comment:
It is hard to imagine a realistic use-case since the original path was absolute. But in the new implementation somebody may override `ManagedWindowsServiceLauncher` and to provide his custom config. Though I am not sure it is sane